src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -58,6 +58,46 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_test_test_case) {
     test_case.run();
 }
 
+OPENVINO_TEST(${BACKEND_NAME}, align_relu_opset_6) {
+    auto model = ngraph::onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "op/relu.hpp"));
```

Comment:
It is wrong, you should prepare *.prototxt files to use a converted *.onnx files in such place.